config/config.txt

```diff
@@ -595,3 +595,7 @@ CACHE_ASSETS 0
 ## You should only enable this if you can guarantee a station name doesn't break BYOND's TOS.
 ## BYOND staff have gotten upset at us in the past for inappropriate station names.
 #STATION_NAME_IN_HUB_ENTRY
+
+## The number of unique PR announcements allowed per round
+## This makes sure that a single person can only spam 3 reopens and 3 closes before being ignored
+PR_ANNOUNCEMENTS_PER_ROUND 5
```

Comment:
Do not make config changes in this pr

Reply:
Fixed with https://github.com/tgstation/tgstation/pull/63879/commits/c499248aba61bb529de3a4ae79b1a26992d08627